README.md

```diff
@@ -8,89 +8,51 @@ TODO: Need correct marketplace ID
 
 This extension provides Puppet Language support for [Visual Studio Code](https://code.visualstudio.com/)
 
-## Platform support
+The extension is composed of the VS Code Extension in the [`client/`](client) directory and the Puppet Language Server in the [`server/`](server) directory
 
-- Microsoft Windows
-- MacOSX
 
-## Features
+## How to run the client and server for development
 
-- [Syntax highlighting](#Syntax_Highlighting)
-- [Code snippets](#Code_snippets)
-- IntelliSense for resources, parameters and more
-- Import from `puppet resource` directly into manifests
-- Node graph preview
+### Run the language server
 
-![Example of features](docs/assets/language_server.gif)
+Follow the instructions in the server documentation - [How to run the Language Server for Develoment](server/READMEmd#How_to_run_the_Language_Server_for_Development). Ensure you use the `--timeout=0` and `--no-stop` arguments so the server does not stop.
```

Comment:
This is in the `How to run the client and server for development` section so it seems applicable.  Perhaps I need to make this more obivious?